src/lib/tooltip/tooltip.ts

```diff
@@ -392,6 +426,113 @@ export class MdTooltip implements OnDestroy {
       this._tooltipInstance._markForCheck();
     }
   }
+
+
+  /**
+   * Registers the tooltip message for accessibility. If this message has not yet been
+   * registered, a new element will be created in a visually hidden container with the message
+   * as its content. This provides screenreaders a reference the tooltip trigger's aria-describedby.
+   * If an element has already been created for this message, increase that registered message's
+   * reference count.
+   */
+  private _registerA11yMessage() {
+    if (!this.message) { return; }
+
+    const a11yMessage = a11yMessages.get(this.message);
+    if (a11yMessage) {
+      a11yMessage.count++;
+    } else {
+      this._createA11yMessageElement(this.message);
+    }
+
+    this._setAriaDescribedBy();
+  }
+
+  /**
+   * Removes the a11y tooltip message element if no other tooltips are registered with the same
+   * message. Otherwise, decrease the reference count of the registered a11y tooltip message.
+   */
+  private _unregisterA11yMessage(message: string) {
+    if (!this._platform.isBrowser || !this.message) { return; }
+
+    // Remove the a11y message if this was its last unique instance.
+    const a11yMessageElement = a11yMessages.get(message);
+    if (a11yMessageElement && --a11yMessageElement.count == 0) {
+      this._deleteA11yMessageElement(message);
+    }
+
+    // If the global messages container no longer has any children, remove it.
+    if (!this._getA11yMessagesContainer()!.childNodes.length) {
+      document.body.removeChild(this._getA11yMessagesContainer()!);
+    }
+  }
+
+  /**
+   * Creates a new element in the visually hidden a11y message container element with the message
+   * as its content.
+   */
+  private _createA11yMessageElement(message: string) {
+    if (!this._platform.isBrowser) { return; }
+
+    // Create a visually hidden container for the accessibility messages if one does not yet exist.
+    if (!this._getA11yMessagesContainer()) {
+      this._createA11yMessagesContainer();
+    }
+
+    const messageElement = document.createElement('div');
+    messageElement.id = `md-tooltip-message-${latestA11yMessageId++}`;
+    messageElement.innerHTML = message;
```

Comment:
This looks a little dangerous. Perhaps we should use `textContent` instead since the spec doesn't allow rich text inside tooltips, IIRC?